scipy/integrate/_bvp.py

```diff
@@ -819,6 +818,10 @@ def solve_bvp(fun, bc, x, y, p=None, S=None, fun_jac=None, bc_jac=None,
             * 0 (default) : work silently.
             * 1 : display a termination report.
             * 2 : display progress during iterations.
+    bc_tol : float, optional
+        Desired absolute tolerance for the boundary condition residuals: `bc` 
+        value should satisfy ``abs(bc) < bc_tol`` component-wise. Equals to
+        `tol` by default.
```

Comment:
Can you find a way to mention the maximum number of iterations for BC residuals convergence?

Reply:
Added the sentence `Up to 10 iterations are allowed to achieve this
        tolerance.` to the end of the description of `bc_tol`